packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6562,6 +6560,61 @@ class RepaintBoundary extends SingleChildRenderObjectWidget {
 /// the semantics layer (and thus e.g. accessibility tools). If
 /// [ignoringSemantics] is null, it uses the value of [ignoring].
 ///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=qV9pqHWxYgI}
+///
+/// {@tool dartpad --template=stateful_widget_material}
+/// The following sample has an [IgnorePointer] widget wrapping the `Column`
+/// which contains a button.
+/// When [ignoring] is set to `true` anything inside the `Column` can
```

Comment:
Is this intended to be a new line, or just text wrap? If it's a new line, you should add an empty line. Single empty lines are ignored.

Reply:
just text wrap